### DISCUSSION TITLE
How To Display Temperature Entity in Climate Button

### DISCUSSION THREAD

**Discussion by bigeazy000** [matt8707/ha-fusion#373] [Q&A] [answered]
First, THANK YOU for HA-Fusion! I'm trying to figure out how to display an entity on a button. For example, I have a Nest thermostat, and it has separate entities for climate and temperature. If I create a button to display the temperature, it works fine, but I have no way to control the thermostat. If I create a button using the climate entity, I can click it and change the temperature, but the main dashboard doesn't display temperature but instead shows whether the HVAC is running (ex. idle). What I would like to do is have the main button display the current temperature from the temperature entity, and optionally have the button change color based on heating, cooling. Is this something that is doable with the current state of HA-Fusion?

**Comment by matt8707** [comment, marked as answer]
once this is implemented https://github.com/matt8707/ha-fusion/issues/381

**Reply by bigeazy000** [reply]
Perfect, I'll keep an eye out. Thanks again for this, its working great for me!
